libs/designer-ui/src/lib/editor/base/utils/editorToSegement.ts

```diff
@@ -31,6 +31,8 @@ const getChildrenNodesToSegments = (node: ElementNode, segments: ValueSegment[],
 };
 
 export const convertStringToSegments = (value: string, tokensEnabled?: boolean, nodeMap?: Map<string, ValueSegment>): ValueSegment[] => {
+  if (!value) return [];
+  if (typeof value !== 'string') return [{ id: guid(), type: ValueSegmentType.LITERAL, value: JSON.stringify(value) }];
```

Comment:
If the `value` param is typed as string do we need to do the JSON.stringify?  If we're not sure what value is we might want to do a try catch or something in case it errors

Reply:
I think it puts proper quotes around it if it wasn't already there:
![CleanShot 2023-07-17 at 12 16 08@2x](https://github.com/Azure/LogicAppsUX/assets/13208452/dc35f78a-d9e3-4da6-91f8-f2896a5fb2bd)

Reply:
Not sure if it's needed but it does make a real difference in output

Reply:
I had tested with some sample values and it seems to be working without it, so I went ahead and removed it